OrbitCore/TracepointCustom.h

```diff
@@ -28,4 +28,8 @@ using TracepointInfoSet =
     absl::flat_hash_set<orbit_grpc_protos::TracepointInfo, internal::HashTracepointInfo,
                         internal::EqualTracepointInfo>;
 
+using TracepointInfoToHashSet =
+    absl::flat_hash_map<orbit_grpc_protos::TracepointInfo, absl::flat_hash_set<uint64_t>,
+                        internal::HashTracepointInfo, internal::EqualTracepointInfo>;
+
 #endif  // ORBIT_TRACEPOINTCUSTOM_H
```

Comment:
The include guard of this file should be `ORBIT_TRACEPOINT_CUSTOM_H_`.

Reply:
Thanks for pointing that out!

Reply:
Sorry, I meant `ORBIT_BASE_TRACEPOINT_CUSTOM_H_` (see the other header files).